CMakeLists.txt

```diff
@@ -426,10 +436,7 @@ PRIVATE
 )
 target_include_directories(
     ktx
-PUBLIC
-    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/lib>
-    $<INSTALL_INTERFACE:lib>
-
+PRIVATE
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/lib/basisu>
     $<INSTALL_INTERFACE:lib/basisu>
```

Comment:
Is this line really necessary? `lib/basisu` is never installed.

Reply:
Same thing as with the transcoder headers, if they are not installed they must not be specified as being part of the interface.